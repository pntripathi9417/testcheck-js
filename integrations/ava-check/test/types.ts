import test from 'ava';
import { gen, check } from '../';

test('generates', check(gen.int, gen.string, (t, x, y) => {
  t.true(x * y === 321)
}))
