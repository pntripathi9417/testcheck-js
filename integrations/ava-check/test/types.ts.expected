test/types.ts(5,14): error TS2363: The right-hand side of an arithmetic operation must be of type 'any', 'number' or an enum type.
